### ISSUE TITLE
Automatic replay

### ISSUE THREAD

**Issue by alexwarren** [textadventures/quest#227]
If you play a game through and spot a mistake, it would be good to be able to
jump into the editor and then go back and replay the game up to that point.  
  
You can already do this of course if you were recording a walkthrough at a
time, but if you're just doing ad hoc testing you might not have recorded one.  
  
What would be nice is an option to (perhaps temporarily) record all of the
moves so far as a walkthrough, even if we weren't recording. We could
basically just always record a walkthrough, then have a menu item to save the
walkthrough when going (back) to the editor.  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 853](http://quest.codeplex.com/workitem/853)
**Status:**	Proposed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Aug 6, 2011 at 11:50 AM
**Reported by:**	alexwarren
**Updated on:**	Jan 6, 2013 at 7:24 PM
**Updated by:**	alexwarren

**Comment by alexwarren** [comment]
No big demand for this, and the player and editor are becoming separate apps in 6.0.
